src/pythermalcomfort/shared_functions.py

```diff
@@ -1,6 +1,163 @@
 import numpy as np
+import pandas as pd
+import re
 
 
 def valid_range(x, valid):
     """Filter values based on a valid range."""
     return np.where((x >= valid[0]) & (x <= valid[1]), x, np.nan)
+
+def convert_and_print_local_clo_values_from_csv_to_dict(csv_name):
```

Comment:
since this is a function that is only used by the JOS model maybe we could include it in one of the files in the `jos3_functions` folder